src/api/spec/cassettes/Staging_StagedRequestsController/POST_create/with_valid_staging_project/1_2_5_4.yml

```diff
@@ -734,106 +25,21 @@ http_interactions:
-    body:
-      encoding: UTF-8
-      string: ''
-    headers:
-      Content-Type:
-      - application/octet-stream
-      Accept-Encoding:
-      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
-      Accept:
-      - "*/*"
-      User-Agent:
-      - Ruby
-  response:
-    status:
-      code: 200
-      message: OK
-    headers:
-      Content-Type:
-      - text/xml
-      Content-Length:
-      - '351'
-      Cache-Control:
-      - no-cache
-      Connection:
-      - close
-    body:
-      encoding: UTF-8
-      string: |
-        <sourcediff key="6b2d3c8abf92970bf1ca6eb54ebee832">
-          <old project="home:permitted_user:Staging:A" package="target_package" rev="0" srcmd5="d41d8cd98f00b204e9800998ecf8427e" />
-          <new project="home:permitted_user:Staging:A" package="target_package" rev="1" srcmd5="8e451641a827df5f4f409d1d543fdb7b" />
-          <files />
-          <issues>
-          </issues>
-        </sourcediff>
-    http_version: 
-  recorded_at: Mon, 01 Apr 2019 14:19:48 GMT
-- request:
-    method: post
-    uri: http://backend:5352/source/home:permitted_user:Staging:A/target_package?cmd=linkdiff&linkrev=base&onlyissues=1&view=xml
-    body:
-      encoding: UTF-8
-      string: ''
-    headers:
-      Content-Type:
-      - application/octet-stream
-      Accept-Encoding:
-      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
-      Accept:
-      - "*/*"
-      User-Agent:
-      - Ruby
-  response:
-    status:
-      code: 200
-      message: OK
-    headers:
-      Content-Type:
-      - text/xml
-      Content-Length:
-      - '398'
-      Cache-Control:
-      - no-cache
-      Connection:
-      - close
-    body:
-      encoding: UTF-8
-      string: |
-        <sourcediff key="43ce61ca22699ae71e20ad86c0705435">
-          <old project="source_project" package="source_package" rev="d41d8cd98f00b204e9800998ecf8427e" srcmd5="d41d8cd98f00b204e9800998ecf8427e" />
-          <new project="home:permitted_user:Staging:A" package="target_package" rev="7eae9f6b12a2579550692e1b40dfc7d6" srcmd5="7eae9f6b12a2579550692e1b40dfc7d6" />
-          <files />
-          <issues>
-          </issues>
-        </sourcediff>
-    http_version: 
-  recorded_at: Mon, 01 Apr 2019 14:19:48 GMT
+  recorded_at: Fri, 18 Oct 2019 10:35:42 GMT
 - request:
     method: put
-    uri: http://backend:5352/source/home:permitted_user/_project/_staging_workflow?user=permitted_user
+    uri: http://backend:5352/source/home:permitted_user:Staging:A/target_package/_link?user=permitted_user
     body:
       encoding: UTF-8
-      string: |
-        <workflow project="home:permitted_user" managers="staging-workflow-managers-11">
-          <staging_project name="home:permitted_user:Staging:A"/>
-          <staging_project name="home:permitted_user:Staging:B"/>
-        </workflow>
+      string: <link project="source_project" package="source_package" rev="d41d8cd98f00b204e9800998ecf8427e"
+        vrev=""></link>
```

Comment:
you wondered if there it's possible to have empty vrevs, didn't you? Looks like you even got a test case for it